server/message_hooks.go

```diff
@@ -38,27 +38,32 @@ func (p *Plugin) UserWillLogIn(_ *plugin.Context, user *model.User) string {
 }
 
 func (p *Plugin) MessageHasBeenPosted(_ *plugin.Context, post *model.Post) {
+	p.GetMetrics().ObserveMessageHooksEvent("message_create")
+	_ = p.syncMessage(post)
```

Comment:
Can we handle this error and at least log it?
```suggestion
	if err := p.syncMessage(post); err != nil {
		p.API.LogWarn("Failed to handle MessageHasBeenPosted", "postID", post.ID, "error", err.Error())
	}
```

Reply:
Good point